core/src/chiplets/hasher.rs

```diff
@@ -64,42 +67,42 @@ pub const TRACE_WIDTH: usize = NUM_SELECTORS + STATE_WIDTH + 2;
 pub const LINEAR_HASH: Selectors = [Felt::ONE, Felt::ZERO, Felt::ZERO];
 /// Unique label for the linear hash operation. Computed as 1 more than the binary composition of
 /// the chiplet and operation selectors [0, 1, 0, 0].
-pub const LINEAR_HASH_LABEL: Felt = Felt::new(3);
+pub const LINEAR_HASH_LABEL: u8 = 3;
```

Comment:
I propose making this update to the BitwiseLookup and MemoryLookup as well (in a separate PR)

Reply:
Agreed!